### PR TITLE
Run device plugin gpu tests on the staging GKE environment against the latest version.

### PR DESCRIPTION
**Pull request by mindprince**

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -6341,6 +6341,26 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gke-staging-latest-device-plugin-gpu": {
+    "args": [
+      "--check-leaked-resources",
+      "--deployment=gke",
+      "--extract=gs://kubernetes-release-gke/release/v1.8.2-gke.0",
```
Comment (krzyzacy) [inline jobs/config.json]: you can use `--extract=ci/gke-latest-staging` now, as https://github.com/kubernetes/test-infra/blob/master/jobs/config.json#L4980

Comment (krzyzacy) [reply jobs/config.json]: I'll update other jobs

**Comment by krzyzacy** [comment]
and you need to rebase now :-) otherwise lgtm

File: jobs/config.json (hunk 1 above)
Comment (mindprince) [reply jobs/config.json]: Done.

**Comment by mindprince** [comment]
Done.

**Pull request merged**